src/style.css

```
@@ -0,0 +1,42 @@
+/* #region Page */
+body {
+  margin: 0;
+  padding: 0;
+  display: block;
+}
+/* #endregion */
+
+/* region Stars */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
```

```suggestion
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```
do not set margin-right for last star.

---

Ok